src/helpers/vote-results.ts

```diff
@@ -57,10 +48,9 @@ export function loadProposalAndSaveVoteResults(
       // assign voting data
       vote.nbYes = voteResults.value0;
       vote.nbNo = voteResults.value1;
-      // vote.onChain = proposal.id; // maybeProposalId;
 
-      vote.voteType = "ONCHAIN";
-      // vote.proposal = proposal.id; // maybeProposalId; //maybeProposalId; // proposalId.toHex();
+      vote.adapterName = "VotingContract";
```

Comment:
I'd just set the value using the variable you already have (`vote.adapterName = votingAdapterName;`) so there's no typo's.